File: spring-security/src/main/java/com/example/presentation/controller/HelloController.java
package com.example.presentation.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloController {

    @RequestMapping("/hogehoge")
    public String helloworld() {
        return "Hello World.";
    }
}

File: kotlin-multi-projects/web/src/main/kotlin/com/example/web/HelloController.kt
package com.example.web

import com.example.core.HelloWorld
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController {

    @RequestMapping("/hello")
    fun hello(): String {
        val helloWorld = HelloWorld()
        return helloWorld.message()
    }
}
